### ISSUE TITLE
Retina Support

### ISSUE THREAD

**Issue by v4n** [LedgerHQ/ledger-wallet-ethereum-chrome#13]
Use SVGs or 2x images in order for the images to display properly in a retina display:
![1u9234](https://user-images.githubusercontent.com/6045161/30525244-d2c12488-9bb7-11e7-87f3-e971fb2e7fe9.png)

**Comment by owenvoke** [comment]
I agree that SVG images would look a lot nicer than the current images. :+1:
